python-avd/pyavd/_eos_designs/schema/schema_fragments/snmp_settings.schema.yml

```
@@ -84,20 +84,37 @@ keys:
         default: false
         description: |
           Generate a local engineId for SNMP using the 'compute_local_engineid_source' method.
+      compute_local_engineid_rfc3411:
+        type: bool
+        default: false
+        description: |
+          Generate a local engineId for SNMP using the 'compute_local_engineid_source' method.
       compute_local_engineid_source:
         type: str
         default: hostname_and_ip
         description: |
           `compute_local_engineid_source` supports:
-          - `hostname_and_ip` generate a local engineId for SNMP by hashing via SHA1
+          - `hostname_and_ip` generate a local engineId for SNMP by
+          If compute_local_engineid_rfc3411 is set to false (default), hashing via SHA1
             the string generated via the concatenation of the hostname plus the management IP.
             {{ inventory_hostname }} + {{ switch.mgmt_ip }}.
+          If compute_local_engineid_rfc3411 is set t True, it will use the default_mgmt_method to find the mgmt ip and calculate an RFC3411 compliant EngineID based on 8000757104+ sha1(hostname + mgmt_ip)
           - `system_mac` generate the switch default engine id for AVD usage.
             To use this, `system_mac_address` MUST be set for the device.
             The formula is f5717f + system_mac_address + 00.
+          - `hostname_inband_ip` generate a local engineId for SNMP by hashing via SHA1
+            the string generated via the concatenation of the hostname plus the management IP, with an RFC3411 compliant prefix.
+            8000757104 + sha1({{ inventory_hostname }} + {{ mgmt_ip }}).
```

Maybe we shoud say inband here though it could be confusing
```suggestion
            "8000757104" + sha1({{ inventory_hostname }} + {{ inband_mgmt_ip }}).
```